projects/helm/chartmuseum/README.md

```diff
@@ -0,0 +1,5 @@
+## **chartmuseum**
+![Version](https://img.shields.io/badge/version-v0.3.7-blue)
+![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiZVJQQTc0Vk8rcHlMR0hOYnllRGNmV0NsQTNLNGFaS2hLME1MUmgwYkxpVUFoL0V0WHZzbXVCV1owQ0FUTlF6RHg1WXhWRXZLRzNwN2d2LzZGUVJvZ0pRPSIsIml2UGFyYW1ldGVyU3BlYyI6Im9jQmZMa216aHZpYmdrWDYiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)
+
+The [chartmuseum project](https://github.com/helm/chartmuseum) is an open-source Helm Chart Repository server written in Go (Golang), with support for cloud storage backends, including Google Cloud Storage, Amazon S3, Microsoft Azure Blob Storage, Alibaba Cloud OSS Storage, Openstack Object Storage, Oracle Cloud Infrastructure Object Storage, Baidu Cloud BOS Storage, Tencent Cloud Object Storage, Netease Cloud NOS Storage, DigitalOcean Spaces, Minio, and etcd.
```

Comment:
Unless Harbor doesn't work without this thing running, we really don't need a helm chart repo for our initial release. Currently, we store our helm charts in a container image registry.